comm_norms.adoc

```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
 As the number of free and open source software projects has increased over time, the communication tools projects use—both to connect with their users and to coordinate their developers and contributors—have evolved.
 Internet Relay Chat (IRC) was once the default choice for real-time project chat.
 Many large projects conducted development discussions solely via their electronic mailing lists, https://en.wikipedia.org/wiki/Freecode[Freshmeat] was the place to hear about new project releases or updates to existing projects, and discussions about feature design and project improvements occurred via email and were included in a project's mailing list archives.
+PD--The sentence in line 14 is quite long. Consider breaking into two sentences (most likely between "projects" and "discussion").
```

Comment:
I agree with this change.

Reply:
Fixing this and any following items that remain to do by i) getting them into a single PR right now, and ii) resolving this conversation as a sign that I've made the change. I won't comment on each one from here out; a closing of a conversation means the same thing from me -- it's been checked as already done, or I just did it.